app/src/test/java/mozilla/lockbox/adapter/ListAdapterTestHelper.kt

```diff
@@ -22,7 +22,7 @@ class ListAdapterTestHelper {
     private val toggleObserverFake = TestObserver<Boolean>()
     private val toggleConsumerFake = TestConsumer(toggleObserverFake) as Consumer<Boolean>
     private val textClicksObserverFake = TestObserver<Unit>()
-    private val textClicksConsumerFake = TestConsumer(toggleObserverFake) as Consumer<Unit>
+    private val textClicksConsumerFake = TestConsumer(textClicksObserverFake)
```

Comment:
Can someone confirm that this change doesnt break semantics?

Reply:
Great catch! Copypasta mistake ☠️